docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -26,23 +30,44 @@ since it must execute these commands in order to create transient users:
   - `visudo`
 - (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/3. Configure a role
+## Step 1/3. Configure RBAC
 
-First, create a role with `create_host_user_mode` set to `keep`. 
-The following role specification will allow users to log in as `nginxrestarter` on
-any matching Node. The host user will be created and added to the groups listed in
-`host_groups`. They will also be given permission to restart the Nginx service as
-root.
+In this section, you will configure a Teleport role that enables host user
+creation.  
 
-<Admonition type="note">
-  `create_host_user_mode` can also be set to `insecure_drop`, which deletes users
-  once the session ends. However, in this mode it is possible for a created user 
-  to get the same UID as a previously deleted user, which would give the new user access
-  to all of the old user's files if they are not deleted. Use `keep` mode
-  unless you really need users to be removed.
-</Admonition>
+When a Teleport user accesses an SSH Service instance, Teleport checks each of
+the user's roles that match the instance. If at least one role matches the
+instance but does not set `create_host_user_mode`, automatic user creation will
+be disabled. Roles that do not match the server will not be checked.
+
+To enable host user creation, you will:
+
+- Label your server so you can match it with a Teleport role that enables host
+  user creation.
+- Create a role that enables host user creation for servers with the label you
+  added.
+- For the purpose of this guide, create a Teleport user with only the role you
+  created.
+
+### Label your server
+
+1. Access the server you enrolled with Teleport before beginning this guide and
+   open the Teleport configuration file, which is `/etc/teleport.yaml` by
+   default.
+
+1. Make the following change to the configuration file:
+
+   ```diff
+     labels:
+   +   host_user_creation: "enabled"
```

Comment:
It's a little strange IMO to use a label like this, it may give an impression you need to have a label with this special name set to enable the feature. Maybe use a more "realistic" label example, e.g. `app: nginx` or something?